### PR TITLE
Release/0.7.1

### PR DESCRIPTION
**Pull request by rzats**

### CODE REVIEW

**Comment by alexanderdean** [comment]
@mhadam can you review this for Rostyslav please?

**Comment by alexanderdean** [comment]
Nudge @mhadam

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 0.7.1 (2018-10-04)
```
Comment (mhadam) [inline CHANGELOG]: this needs to be changed to today

File: sql_runner/main.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ import (
 const (
 	CLI_NAME        = "sql-runner"
 	CLI_DESCRIPTION = `Run playbooks of SQL scripts in series and parallel on Redshift and Postgres`
-	CLI_VERSION     = "0.7.0"
+	CLI_VERSION     = "0.7.1-rc1"
```
Comment (mhadam) [inline sql_runner/main.go]: change this to 0.7.1

**Pull request merged**